assets/section-footer.css

```diff
@@ -282,6 +300,10 @@
   }
 }
 
+.localization-form .disclosure {
+  display: flex;
+}
```

Comment:
Curious as to why this is needed as well. Seem like that element contains the button and the container showing when the button is clicked but since it's absolutely positioned I'm not too sure what it does 🤔